src/pages/Article.jsx

```diff
@@ -24,6 +27,21 @@ function Article() {
         })
     }, [article_id])
 
+
+
+    useEffect(() => {
+        getCommentsData(article_id)
+        .then((data) => {
+            setCommentsLoading(false)
+            setComments(data.comments)
+        })
+        .catch((err) => {
+            console.error("error fetching the comment", err)
+            setCommentsLoading(false)
+        })
+    }, [article_id])
```

Comment:
it's unnecessary to have another `useEffect` for this code, it can all be executed in the scope of the same `useEffect` you have above by chaining promises